packages/sources/react/package.json

```diff
@@ -19,9 +19,10 @@
   },
   "sideEffects": false,
   "license": "Apache-2.0",
-  "main": "dist/index.js",
-  "module": "dist/index.es.js",
-  "jsnext:main": "dist/index.es.js",
+  "type": "module",
```

Comment:
@Shyrro could this be a breaking change for consumers using node 14 and/or the require syntax? Do we have a mean to know if we have such users?

Reply:
@thibault-mahe Uhm, i don't know if we have such users, but the `type: module` just indicated that all the files that end with `.js` are `esmodules` by default, if someone uses `require`, the target becomes the main and uses the `cjs` export instead. 
I don't think it breaks on node 14 but would be interested to test it